### ISSUE TITLE
Cloning a revision from a template revision with latest K8s versions gets saved without the satisfies condition

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#23415]
Rancher Server Version - v2.3.1-rc2

Steps :
1. Create a Template Revision R1 with K8s version set to Latest 1.15 (or Latest 1.16 or Latest 1.14 or Latest 1.13)
Do not set override. This revision gets saved with questions and the satisfies condition is set in the template as below
```
"questions": [
{
"default": "1.15.x",
"max": 0,
"maxLength": 0,
"min": 0,
"minLength": 0,
"required": false,
"satisfies": ">=1.15.0 <1.16",
"type": "string",
"variable": "rancherKubernetesEngineConfig.kubernetesVersion"
}
],

```
2. Create another revision R2 by cloning this template revision R1(OR creating a new revision from Default). This revision gets saved as  a revision with question but the satisfies condition does not exist any more.
```
"questions": [
{
"default": "1.15.x",
"max": 0,
"maxLength": 0,
"min": 0,
"minLength": 0,
"required": false,
"type": "string",
"variable": "rancherKubernetesEngineConfig.kubernetesVersion"
}
],

```

**Comment by westlywright** [comment]
Available in UI tag `2.3.17`

**Comment by soumyalj** [comment]
Tested with rancher v2.3.1-rc2 with the UI tag 2.3.17
Case 1:
1. Create a Template Revision R1 with K8s version set to Latest 1.16 without setting override.  This revision gets saved with questions and the 'satisfies' condition is set in the template. 
2. Create another revision R2 by cloning this template revision R1(OR creating a new revision from Default). This revision gets saved as a revision with question and the 'satisfies' condition is set correctly as below:
```
questions": [
{
"default": "1.16.x",
"max": 0,
"maxLength": 0,
"min": 0,
"minLength": 0,
"required": false,
"satisfies": ">=1.16.0 <1.17",
"type": "string",
"variable": "rancherKubernetesEngineConfig.kubernetesVersion"
}
]
```
Case 2: 
1. Create a Template Revision R1 with K8s version set to Latest 1.15 without setting override.  This revision gets saved with questions and the 'satisfies' condition is set in the template. 
2. Create another revision R2 by cloning this template revision R1(OR creating a new revision from Default). This revision gets saved as a revision with question and the 'satisfies' condition is set correctly as below:. questions": [
{
"default": "1.15.x",
"max": 0,
"maxLength": 0,
"min": 0,
"minLength": 0,
"required": false,
"satisfies": ">=1.15.0 <1.16",
"type": "string",
"variable": "rancherKubernetesEngineConfig.kubernetesVersion"
}
]
